modules/apps/adaptive-media/adaptive-media-image-impl-test/src/testIntegration/java/com/liferay/adaptive/media/image/internal/scaler/test/AMImageScalerRegistryImplTest.java

```diff
@@ -347,7 +348,7 @@ private void _unregisterAMImageScaler(
 	private AMImageScaler _amDefaultImageScaler;
 
 	@Inject
-	private AMImageScalerTracker _amImageScalerTracker;
+	private AMImageScalerRegistry _amImageScalerRegistry;
```

Comment:
@slnn, could you please merge the commit to https://github.com/liferay-core-infra/liferay-portal/pull/2299/commits/ea94f990ebc22583f63d13e3d641308eabf46b3c. I think we may treat the change as usage. After that, the pr should be more clear(The first commit: rename *Registry and *RegistryImpl; The second commit: Apply usages).  You may push force your change. And then I will approve the pr to Dante review.

Thanks,
Hai

Reply:
Done